app/Http/Controllers/OrderController.php

```
@@ -17,33 +18,42 @@
-            ->when($user->isEmployee(), function ($query) use ($user) {
-                $query->where('assigned_to', $user->id)
-                    ->orWhere('created_by', $user->id);
-            })
-            ->when($user->isAdministrator() || $user->isSuperAdministrator(), function ($query) use ($user) {
-                // No additional query modification needed for administrators
-        })
-            ->get();
-
-        return response()->json(OrderResource::collection($ordersQuery));
+        $key = self::indexKeyFor($user);
+        $hit = Cache::has($key);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'index'. (undefined)

(StaticAccess)